app/components/shared/event-item.tsx

```diff
@@ -1,16 +1,31 @@
 import { Link } from "@remix-run/react"
 
 import { ButtonLink } from "~/components/ui/button-link"
-import { formatPublished } from "~/utils/datetime"
+import {
+  formatPublished,
+  formatPublishedWithTime,
+  formatTime,
+} from "~/utils/datetime"
+import { Iconify } from "../ui/iconify"
```

Comment:
@fikrialwan Need follow convention `~/components/`

Reply:
I've resolve it on this commit https://github.com/bandungdevcom/bandungdev.com/pull/42/commits/21ace63e6a71c64a0fcb22e80e127afb1ef97649